README.md

```diff
@@ -74,7 +74,7 @@ Thus, webpack CLI provides different commands for many common tasks.
 
 ### Utilities
 
-The project also has several utility packages which are used by other commands
+The project also have several utility packages which are used by other commands
```

Comment:
prev is correct

Reply:
Yes, `has` is correct.
![Screenshot at 2020-08-28 18-26-33](https://user-images.githubusercontent.com/46647141/91563186-14982600-e95c-11ea-96d3-d75faefa8445.png)